src/Stratis.FederatedPeg.Features.FederationGateway/FederationGatewayFeature.cs

```diff
@@ -190,15 +195,21 @@ public static class FullNodeBuilderSidechainRuntimeFeatureExtension
     {
         public static IFullNodeBuilder AddFederationGateway(this IFullNodeBuilder fullNodeBuilder)
         {
-            LoggingConfiguration.RegisterFeatureNamespace<FederationGatewayFeature>("federationgateway");
-
+            LoggingConfiguration.RegisterFeatureNamespace<FederationGatewayFeature>(FederationGatewayFeature.FederationGatewayFeatureNamespace);
+            
             fullNodeBuilder.ConfigureFeature(features =>
             {
                 features
                     .AddFeature<FederationGatewayFeature>()
                     .DependOn<BlockNotificationFeature>()
                     .FeatureServices(services =>
                     {
+                        services.AddHttpClient(
+                            FederationGatewayFeature.JsonHttpClientName,
+                            client => {
+                                    client.DefaultRequestHeaders.Accept.Clear();
+                                    client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+                            });

```

Comment:
Now adding the service here, does this make the code in `RestSenderBase.SendAsync()` - lines 32-33 redundant ?